conan/conanfile.txt

```diff
@@ -9,6 +9,7 @@ jsonformoderncpp/3.1.0@vthiery/stable
 graylog-logger/1.0.3-dm1@ess-dmsc/stable
 h5cpp/0.0.5@ess-dmsc/stable
 streaming-data-types/3e85b23@ess-dmsc/stable
+cli11/1.3.0@bincrafters/stable
```

Comment:
Can you add the remote for ``cli11`` in the README ?

Reply:
Jonas already did that on `master`, see https://github.com/ess-dmsc/kafka-to-nexus#conan